math_.py

```diff
@@ -13,3 +13,14 @@ def substract(a: int, b: int) -> int:
 def list_substract(a: List, b: List) -> List:
     assert len(a)==len(b), "len(a) != len(b)"
     return [i-j for i, j in zip(a,b)]
+
+def isnum(a: Any):
+    if isinstance(a, int):
```

Comment:
The `try...except` block is catching a very broad `Exception`. This could potentially hide other errors that you might want to handle differently. Consider making the `except` clause more specific, catching only the types of errors you expect.